webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -4,12 +4,14 @@ import PropTypes from 'prop-types';
 import { translate as __ } from 'foremanReact/common/I18n';
 import { Button, Icon } from 'patternfly-react';
 import { INSIGHTS_SYNC_PAGE_TITLE } from './InsightsCloudSyncConstants';
+import InsightsSettings from './Components/InsightsSettings';
 
 const InsightsCloudSync = ({ data: { settingsUrl }, syncInsights }) => {
   document.title = INSIGHTS_SYNC_PAGE_TITLE;
   return (
     <IntlProvider locale={navigator.language}>
       <div className="insights-cloud-sync">
+        <InsightsSettings />
```

Comment:
Can you move it to float right? (same as in the inventory page)
![Selection_345](https://user-images.githubusercontent.com/26363699/89124110-10075b80-d4dd-11ea-956c-492e788023d1.png)

Reply:
Done